tools/packaging/static-build/qemu/build-base-qemu.sh

```diff
@@ -38,16 +38,15 @@ CACHE_TIMEOUT=$(date +"%Y-%m-%d")
 [ -n "${build_suffix}" ] && HYPERVISOR_NAME="kata-qemu-${build_suffix}" || HYPERVISOR_NAME="kata-qemu"
 [ -n "${build_suffix}" ] && PKGVERSION="kata-static-${build_suffix}" || PKGVERSION="kata-static"
 
-container_image="${QEMU_CONTAINER_BUILDER:-$(get_qemu_image_name)}"
+container_image="${QEMU_CONTAINER_BUILDER:-${BUILDER_REGISTRY}:qemu-$(get_last_modification ${script_dir})-$(uname -m)}"
```

Comment:
I think this needs reverting to the old approach to pick up `CC_BUILDER_REGISTRY` from lib.sh rather than `BUILDER_REGISTRY`
```suggestion
container_image="${QEMU_CONTAINER_BUILDER:-$(get_qemu_image_name)}"
```